client/src/styles/AdminDeleteProduct.css

```diff
@@ -0,0 +1,15 @@
+.product-list-item {
+  margin: 15px;
+  padding: 5px;
+  background-color: #FCD8D4;
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+strong {
+  font-weight:bold;
+ }
```

Comment:
Sorry this is just a small nitpick, but could you adjust the spacing? There should be a space between `font-weight:` and `bold;`, and it also looks like there is a space before the closing `}` curly bracket.

Reply:
Done!